inst/rmarkdown/templates/xaringan/resources/kunoichi.css

```diff
@@ -0,0 +1,144 @@
+@import url(https://fonts.googleapis.com/css?family=Yanone+Kaffeesatz);
+@import url(https://fonts.googleapis.com/css?family=Source+Code+Pro:400,700);
+
+/******************
+ kunoichi.css must also be coupled with ninjutsu.css to be effective
+ See https://emitanaka.github.io/ninja-theme/kunoichi-theme-example.html for an example.
+
+******************/
+
+
+:root {
+    --main-color1: rgba(86,36,87,1);
+    --main-color2: #9C27B0;
+    --main-color3: #BA68C8;
+    --main-color4: #CE93D8;
+    --main-color5: white;
+    --text-color1: white;
+    --text-color2: white;
+    --text-color3: white;
+    --text-color4: white;
+    --text-color5: black;
+    --text-opacity: white;
+    --link-color: #ecadec;
+    --alert-color: #CF3C18;
+    --text-alert: white;
+    --code-background: #f8f8f8;
+    --code-inline-color: #FFD740;
+    --code-text-color: black;
+    --code-highlight: rgba(136, 57, 138, 0.3);
+    --color-opacity70: rgba(86,36,87,0.7);
+    --color-opacity50: rgba(86,36,87,0.5);
+    --color-opacity30: rgba(86,36,87,0.3);
+    --logo: url("../images/R-LadiesGlobal-transparent.png");
+    --title-background: url("../bkgs/bg1.jpg");
```

Comment:
Will these images with relative URLs actually work? They do not seem to be included in the PR. Please feel free to use absolute URLs like `https://emitanaka.github.io/ninja-theme/images/R-LadiesGlobal-transparent.png`.

Reply:
That's a totally a good point! And also that absolute url.. I had no idea you could do that. That's very neat. 
The `shinobi` theme has a similar issue except I left the url empty which I didn't think it was good coding practice but it did what I wanted it to (which is to render nothing) and still making it obvious to the user they need to add their url there. But given this will be a built-in theme I guess it would make more sense if the user defines the images in their own custom css. (Okay I've decided to just remove it except the R Ladies logo). 

Comments to make it better always welcomed! 

Pull request coming.